src/class-wpml-gutenberg-strings-in-block.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+/**
+ * Class WPML_Gutenberg_Strings_In_Block
+ */
+class WPML_Gutenberg_Strings_In_Block {
+
+	/** @var array */
+	private $block_types = array(
+		'core/paragraph' => array( '//p' ),
+		'core/heading'   => array( "//*[name()='h1' or name()='h2' or name()='h3' or name()='h4' or name()='h5' or name()='h6']" ),
```

Comment:
@BruceOnTheGo I think that's a better way to do the same: `//*[self:: h1 or self:: h2 or self::h3 or self::h4 or self::h5 or self::h6]`.

It also looks like your solution may have unpredictable effects in some circumstances: https://stackoverflow.com/a/722251/626538

Reply:
@andreasciamanna I've updated the query in the wpml-config.xml file - https://git.onthegosystems.com/wpml/sitepress-multilingual-cms/merge_requests/1986/diffs